### DISCUSSION TITLE
losing connection during llm model loading or token generation

### DISCUSSION THREAD

**Discussion by linxOD** [open-webui/open-webui#3801] [General]
I loaded the latest docker image v0.3.8:
```bash
podman run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -v /home/llm/ollama/data/lioe:/app/backend/data:z --name open-webui --restart always ghcr.io/open-webui/open-webui:latest
```
Using open webui via localhost worked fine but switching to a reverse proxy had some issues. Some of them I could resolve by reading closed issues like:

* [Nginx Config](https://github.com/open-webui/open-webui/discussions/1235#discussioncomment-9341387)
* [Websocket](https://github.com/open-webui/open-webui/issues/3054#issuecomment-2163426012)

and updating the server config. I also updated the server timeout to 10min.

However, one remaining issue I could no find anything about.
Especially with larger models but not only, I receive this error in the browser console:
`<my-proxy>/ollama/api/chat net::ERR_NETWORK_CHANGED 200 (OK)`
The error appears either when a new model is loading in the ollama container or during token gerneration and the stream just breaks up.

I do not see any errors in the ollama docker container logs and also no errors in the open-webui container logs.

Any Ideas?

**Comment by justinh-rahb** [comment]
`net::ERR_NETWORK_CHANGED` is a Chrome error indicating something about your network connection changed, such as your IP for example. If you have an unstable internet connection where your Chrome-running device is located, this usually is the result. Notably you would not get this error if the issue was the _server's_ connection, only your browser device.

**Comment by linxOD** [comment]
Thank your for your feedback! Firefox had some better results. At least the stream continued even after some websocket connection change. Will try to figure out why my internet connection isn't stable.
